web-app/HeimScripts/heatmap/run.R

```diff
@@ -143,16 +156,20 @@ addStats <- function(df, sorting, ranking, max_rows) {
     df["MEAN"]         <- means
     df["SD"]           <- sdses
     df["SIGNIFICANCE"] <- rankingScore
-    df                 <- applyRanking(df, ranking, max_rows)
+
+    if (useLimma && validLimmaMeasurements) {
```

Comment:
Shouldn't ranking be also done if !useLimma?  Isn't the only case in which ranking shouldn't be applied is if(useLimma & !validLimmaMeasurements)